style.css

```diff
@@ -0,0 +1,305 @@
+body{
+    margin: 0;
+}
+
+html{
+    font-size: 100%;
+    font-family: Open Sans,sans-serif;
+}
+
+#nav-menu{
+    padding-top: 8px;
+    padding-bottom: 8px;
+    position: relative;
+    background: #fff;
+    border-bottom: 1px solid #ebf2f2;
+}
+
+@media (max-width: 1200px) and (min-width: 900px){
+#nav-menu{
+    padding-right: 20px;
+}
+}
+
+@media (max-width: 900px){
+#nav-menu {
+    position: fixed;
+    width: 100%;
+    border-bottom: 1px solid #ebf2f2;
+}
+}
+
+.container{
+    display: flex;
+    max-width: 1200px;
+    padding: 0 20px;
+    margin: 0 auto;
+    align-items: center;
+    justify-content: space-between;
+}
+
+
+#Logo{
+    height: 48px;
+    max-width: 250px;
+}
+
+@media (max-width: 1100px){ 
+    #Logo {
+    height: 30px;
+}
+}
+
+.header_logo{
+    display: block;
+}
+
+#menu_items{
+    display: flex;
+    list-style-type: none;
+    padding: 0;
+    width: 700px;
+    align-items: center;
+    justify-content: space-between;
+    font-weight: 700;
+    font-size: 1rem;
+
+}
+
+@media (max-width: 900px){
+#menu_items {
+    display: none;
+}
+}
+
+.dropdownMenu{
+    transform: rotate(45deg) translateY(-6px);
+    -webkit-transform: rotate(45deg) translateY(-6px);
+    border: solid #191e1e;
+    border-width: 0 1px 1px 0;
+    display: inline-block;
+    padding: 3px;
+    margin-left: 5px;
+}
+
+
+  
+  li {
+    color:inherit;
+    display: block;
+    float: left;
+    padding: 1rem;
+    position: relative;
+    text-decoration: none;
+  }
+    
+  li a {
+    color: inherit;
+    text-decoration: none;
+  }
+  
+  li:hover {
+    cursor: pointer;
+  }
+  
+  ul li ul {
+    visibility: hidden;
+    opacity: 0;
+    position: absolute;
```

Comment:
it is bad practice styling dom elements with specific styles (dropdown styles). Please use class for it instead